content/docs/command-reference/exp/init.md

```diff
@@ -17,37 +18,52 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 ## Description
 
 `dvc exp init` helps you quickly get started with experiments. It reduces
-boilerplate for initializing [pipeline](/doc/command-reference/dag) stages in a
-`dvc.yaml` file by assuming defaults about the location of your data,
-[parameters](/doc/command-reference/params), source code, models,
-[metrics](/doc/command-reference/metrics) and
-[plots](/doc/command-reference/plots), which can be customized through config.
+boilerplate DVC procedures by creating a `dvc.yaml` file that assumes default
+location of your input data, <abbr>parameters</abbr>, source code, models,
+<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
+can be customized through the options of this command or via config files.
+Standard repository structure:
 
-It also offers guided `--interactive` mode for creating a stage to be
-[`exp run`](/doc/command-reference/exp/run) later. `dvc exp init` supports
-creating different types of stages, eg: `dl` if you are doing deep learning,
-which uses [dvclive](/doc/dvclive) to monitor and checkpoint progress during
-training of machine learning models.
+```
+├── data/
+├── metrics.json
+├── models/
+├── params.yaml  # required
+├── plots/
+└── src/
+```
+
+> Note that `params.yaml` is the only required file (see `dvc params`).
```

Comment:
side note ... for the sake of my education: why is it required?

Reply:
To parse the parameters and put them into `dvc.yaml`.

Reply:
Related to https://github.com/iterative/dvc/issues/6446#issuecomment-988284225

Reply:
It's also related to https://github.com/iterative/dvc/issues/6605. The `params` section of `dvc.yaml` currently expects keys from a file (see below), not a filename itself, so all the parameters need to be defined before creating the stage. If we want to stop requiring this file at the time of `dvc exp init`, we need to change `params` to accept filenames.

```
stages:
  train:
    cmd: python train.py
    deps:
    - data
    - src
    params:
    - params.py:
      - batch_size
      - data_path
      - epochs
      - latent_dim
      - num_samples
```

Reply:
What about generating a dvc.yaml without `params` if there's no params.yaml file? This is a core discussion though!

Reply:
Sorry, I misunderstood you, and now I think it's actually more of a docs issue 🤔 . It is possible to use `dvc exp init` without `params`. You can either use `--explicit` and not provide `--params`, or you can use `dvc exp init -i` and type `n` at the params prompt.

The note here is to indicate that if you do use either the default or some other `params` path, then that path must exist and be parseable to extract the parameters for the stage. So the current note probably needs to be reworded.

Reply:
UPDATE: Looks like the discussion about empty params.yaml files is in a few core repo tickets now (e.g. https://github.com/iterative/dvc/issues/7138) so I won't keep discussing the current functionality here (still a bit confused though).

> It is possible to use dvc exp init without params. You can either use --explicit and not provide --params, or you can use dvc exp init -i and type n at the params prompt...
So the current note probably needs to be reworded.

Clarified in 2273431a.